aiida/sphinxext/process.py

```diff
@@ -117,9 +117,12 @@ def build_content(self):
         content += self.build_doctree(title='Outputs:', port_namespace=self.process_spec.outputs)
 
         if hasattr(self.process_spec, 'get_outline'):
-            outline = self.process_spec.get_outline()
-            if outline is not None:
-                content += self.build_outline_doctree(outline=outline)
+            try:
+                outline = self.process_spec.get_outline()
+                if outline is not None:
+                    content += self.build_outline_doctree(outline=outline)
+            except AssertionError:
+                pass
```

Comment:
Why is this necessary?

Reply:
because `get_outline` can no longer return `None` (required by plumpy typing), instead it raises an assertion error if the outline is not yet loaded